moveit_planners/ompl/ompl_interface/src/ompl_planner_manager.cpp

```diff
@@ -295,8 +295,6 @@ class OMPLPlannerManager : public planning_interface::PlannerManager
       pub_valid_states_ = nh_.advertise<moveit_msgs::DisplayRobotState>("ompl_planner_valid_states", 5);
       pub_valid_traj_ = nh_.advertise<moveit_msgs::DisplayTrajectory>("ompl_planner_valid_trajectories", 5);
       display_random_valid_states_ = true;
-      //    pub_valid_states_thread_.reset(new boost::thread(boost::bind(&OMPLPlannerManager::displayRandomValidStates,
-      //    this)));
```

Comment:
Please do not remove this comment. If anything, we should remove the whole commented out blocks instead and that would belong into a separate PR.

Reply:
Here I react to all comments on the removal of commented out code. I thought to remove these commented lines as they are dead and not subject to CI checks. Commented out code is an [official code smel](https://moderatemisbehaviour.github.io/clean-code-smells-and-heuristics/comments/c5-commented-out-code.html). When I encountered these comments I simply deleted them, as I thought that searching for them later is difficult and cumbersome. I will change all comments back and add them to a separate PR.

Reply:
> Commented out code is an official code smell

I agree! However, in these parts it's actually worse than commented out code.
Some of these commented out functions would actually be useful from my perspective but do not apply anymore because of API changes over time.
In other cases, also in this particular one, the actual functionality is commented out, but not the whole related structure.
The DynamicReconfigure option still exists!